Composer/packages/server/src/models/connector/csharpBotConnector.ts

```diff
@@ -109,3 +213,19 @@ export class CSharpBotConnector implements IBotConnector {
     });
   };
 }
+
+process.on('SIGINT', () => {
+  log('[SIGINT] start graceful shutdown');
+  CSharpBotConnector.stopAll('SIGINT');
+  process.exit(1);
```

Comment:
Why are we exiting with an error code? (Also applies to the following)

Reply:
Because exiting with interrupt? You means we can treat this situation (such like ctrl+c) as a normal exit?